tests/testthat/test_convertMT.R

```diff
@@ -0,0 +1,51 @@
+test_that("Migration time conversion works", {
+  
+  ## load test data
+  rtime <- c(10, 20, 30, 40, 50, 60, 70, 80, 90, 100)
+  marker <- data.frame(markerID = c("marker1", "marker2"),
+                       rtime = c(20, 80),
+                       mobility = c(0, 2000))
+  marker_E <- data.frame(markerID = c("marker1", "marker2"),
+                       rt = c(20, 80),
+                       mobility = c(0, 2000))
+  marker_E2 <- data.frame(markerID = c("marker1", "marker2"),
+                         rtime = c(20, 80),
+                         mobility = c("A", "B"))
+  marker_E3 <- data.frame(markerID = c("marker1", "marker2"),
+                         rtime = c("A", "B"),
+                         mobility = c(0, 2000))
+  
+  
+  µ_s1 <- convertMT(rtime, marker[-2,], U = 30, L = 90)
```

Comment:
Please replace the greek letters here with e.g. `m`, i.e. that you have variables `m_s1` etc. Might be that this works on all systems and all language settings, but I'm not completely sure.